src/components/layout/Layout.module.scss

```diff
@@ -16,15 +16,16 @@
   background-attachment: fixed;
   height: 100vh;
```

Comment:
min-height: 100vh 
because if there is more content, it will go beyond the boundaries

Reply:
I don't need to go beyond edges.